### PR TITLE
Add new design for OperationsList

### PR DESCRIPTION
**Pull request by loeck**

### CODE REVIEW

File: src/components/OperationsList/index.js [hunk 1]
```diff
@@ -91,89 +78,124 @@ const ShowMore = styled(Box).attrs({
...
+
+const Address = ({ value }: { value: string }) => {
+  const addrSize = value.length / 2
+
+  const left = value.slice(0, 10)
+  const right = value.slice(-addrSize)
+  const middle = value.slice(10, -addrSize)
+
+  return (
+    <Box horizontal color="smoke" ff="Open Sans" fontSize={3}>
```
Comment (gre) [inline src/components/OperationsList/index.js]: should `Open Sans` be in a constant? same Q for `smoke` 🤔

File: src/components/OperationsList/index.js [hunk 2]
```diff
@@ -91,89 +78,124 @@ const ShowMore = styled(Box).attrs({
...
+  t: T,
   tx: OperationType,
   withAccount?: boolean,
-  minConfirmations: number,
 }) => {
-  const acc = account || tx.account
+  const { unit } = account
   const time = moment(tx.receivedAt)
-  const Icon = getIconByCoinType(get(tx, 'account.currency.coinType'))
+  const Icon = getIconByCoinType(get(account, 'currency.coinType'))
```
Comment (gre) [inline src/components/OperationsList/index.js]: why not doing `account.currency.coinType` ? We should try to limit nb of usage of `get` to not lose the (flow) type

File: src/components/OperationsList/index.js [hunk 3]
```diff
@@ -240,43 +272,40 @@ class OperationsList extends Component<Props> {
...
-                minConfirmations={minConfirmations}
-                tx={tx}
-              />
-            ))}
-          </Box>
+        <Box>
+          <Card flow={1} title={title} p={0}>
+            <Box>
+              {operations.map(tx => {
+                const acc = account || tx.account
```
Comment (gre) [inline src/components/OperationsList/index.js]: what is this `account || tx.account` ? 🤔

File: src/components/OperationsList/index.js (hunk 1 above)
Comment (loeck) [reply src/components/OperationsList/index.js]: Actually, it's a constant :D
See: https://github.com/LedgerHQ/ledger-wallet-desktop/blob/master/src/styles/theme.js

File: src/components/OperationsList/index.js (hunk 3 above)
Comment (loeck) [reply src/components/OperationsList/index.js]: You can have `account` in 2 ways, so, in operation object (for example, in Dashboard https://github.com/LedgerHQ/ledger-wallet-desktop/blob/master/src/components/DashboardPage/index.js#L67 when we want to display OperationsList with different accounts) and with props (in AccountPage (https://github.com/LedgerHQ/ledger-wallet-desktop/blob/master/src/components/AccountPage/index.js#L160 we pass directly props)

File: src/components/OperationsList/index.js (hunk 2 above)
Comment (loeck) [reply src/components/OperationsList/index.js]: Yup, you're right, I fix that

File: src/components/OperationsList/index.js (hunk 3 above)
Comment (gre) [reply src/components/OperationsList/index.js]: oh yeah I see. makes sense. I guess later we will have to change that if tx.account is gone. like `account || findInAccounts(accounts, tx.accountId)`,

File: src/components/OperationsList/index.js (hunk 1 above)
Comment (gre) [reply src/components/OperationsList/index.js]: holy smoke :D 👍

**Pull request merged**